make/RunTestsPrebuiltSpec.gmk

```diff
@@ -176,3 +176,7 @@ ULIMIT := ulimit
 ifeq ($(OPENJDK_BUILD_OS), windows)
   PATHTOOL := cygpath
 endif
+
+ifneq ($(TIDY_HOME), )
+  TIDY := $(TIDY_HOME)
+endif
```

Comment:
This is worse than before. Please forget what I said initially about `TIDY_HOME`, it should certainly not be used as an alias for `TIDY`.

Reply:
I may need suggestions, I pushed the last changes to the other PR [f5aeddd](https://github.com/openjdk/jdk/pull/21272/commits/f5aeddd8a2b142934558d75b5b81823ddeefd6e0).

I need a way to set `TIDY` without using configure, so that `-Dtidy` is set in RunTests.gmkl.

Maybe I can change `testOnlyProfilesPrebuiltDocs["run-test-prebuilt-docs"].environment["TIDY_HOME"] = input.get("tidy", "home_path") + "/bin/tidy";` to use `TIDY` instead of `TIDY_HOME`? There would be no conflict as configure is not run and `TIDY=` was never passed

Reply:
That is exactly what I would like you to do.

Reply:
The two different but yet closely related PRs are making this hard to review. I thought I saw the changes to `testOnlyProfilesPrebuiltDocs` but then it was not there. Until I realized that that tidy change is in the other PR. 

That is not really ideal. Otoh, for this bug to be integrated first, it cannot rely on structs that are only created in the second PR. So I guess that is okay, for now. In the end, neither this nor the docs-test root PR will achieve much on their own, but is only laying the groundwork.